app/retail/templates/shared/head.html

```diff
@@ -61,7 +61,7 @@
 <title>{% if title %}{{title}}{% else %}{% trans "Grow Open Source" %}{% endif %} | Gitcoin</title>
 {% include 'shared/favicon.html' %}
 <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
-
+<!-- <script src="https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.0.0-beta.34/dist/web3.min.js"></script> -->
```

Comment:
should be commented out?

Reply:
So first I started injection only if not metamask present but I believe we should inject always and stop using metamask injected one. In that way we can used default methods and avoid having special code just for metamask.